source/whats-new.txt

```diff
@@ -0,0 +1,54 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 4.2 <version-4.2>`
+* :ref:`Version 4.1 <version-4.1>`
+* :ref:`Version 4.0 <version-4.0>`
+
+
+.. _version-4.2:
+
+What's New in 4.2
+-----------------
+New features of the 4.2 Java driver release include:
+
+- Enabled Azure and GCP key stores to client-side field level encryption
```

Comment:
Some items in this list are "Enabled" and others are "Added". Is there a difference? If not, choose 1 for consistency. I think "Added" is better, IMHO.